spk/vim/Makefile

```
@@ -12,11 +13,18 @@ DESCRIPTION_FRE = Vim est un éditeur de texte avancé basé sur Vi \(unix\) ave
 DISPLAY_NAME = Vim
 STARTABLE = no
 
-CHANGELOG = "Update vim to v9.0."
+CHANGELOG = "1. Update vim to v9.0"
 
 HOMEPAGE = https://www.vim.org/
 LICENSE  = Charityware
 
+include ../../mk/spksrc.archs.mk
+
+ifneq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(OLD_PPC_ARCHS)),$(ARCH))
+CHANGELOG += "<br/>2. Migrate to Python 3.11"
+CHANGELOG += "<br/><br/>NOTE: You need to install Python 3.11 package to enable Python support."
+endif
+
```

What about this:
```suggestion
include ../../mk/spksrc.archs.mk

ifneq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(OLD_PPC_ARCHS)),$(ARCH))
CHANGELOG += "<br/>2. Migrate to Python 3.11"
DESCRIPTION += "NOTE: You need to install Python 3.11 package to enable Python support."
else
CHANGELOG += "<br/>2. Remove  python support"
DESCRIPTION += "NOTE: This package is built without Python support."
endif
```

But I don't know whether a user will see the arch dependent changelog.
At least those are not visible in the repo at https://synocommunity.com/package/vim.